src/Views/OperatingSystemView.vala

```diff
@@ -215,8 +247,55 @@ public class About.OperatingSystemView : Gtk.Box {
         updates_list.get_first_child ().focusable = false;
         updates_list.get_last_child ().focusable = false;
 
+        var thebasics_link = new LinkRow (
+            documentation_url,
+            _("Basics Guide"),
+            "text-x-generic-symbolic",
+            "green"
+        );
+
+        var support_link = new LinkRow (
+            support_url,
+            _("Get Community Support"),
```

Comment:
I wonder if this should be "Get Help" instead? Is it just the word "Support" that implies its a support technician? Theoretically a distro could replace this link with a link to some paid support staff, so maybe it's not good to include "Community" here
```suggestion
            _("Get Help"),
```